system-tests/e2e-transfer-test/control-plane-cosmosdb/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+val cosmosSdkVersion: String by project
+
+plugins {
+    `java-library`
+}
+
+dependencies {
+    implementation(project(":system-tests:e2e-transfer-test:control-plane"))
+    implementation(project(":extensions:azure:cosmos"))
+    implementation(testFixtures(project(":extensions:azure:azure-test")))
+    implementation("com.azure:azure-cosmos:${cosmosSdkVersion}")
```

Comment:
could we make the `cosmos-common` extension expose `"com.azure:azure-cosmos` as `api` configuration? then we'd not have to state it here again

Reply:
this was a spiking leftover, not needed, removed